public/js/services/game.js

```diff
@@ -12,7 +12,7 @@ angular.module('mean.system')
     table: [],
     czar: null,
     playerMinLimit: 3,
-    playerMaxLimit: 11,
+    playerMaxLimit: 6,
```

Comment:
Merge Conflict issues. @andela-wmaina changed this to 11, you change it back to 6?